content/en/blog/2025/zombie_members_upgrade.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Avoiding Zombie Cluster Members When Upgrading to etcd v3.6
+author:  "[Benjamin Wang](https://github.com/ahrtr) VMware by Broadcom, [Josh Berkus](https://github.com/jberkus) Red Hat"
+date: 2025-12-17
+draft: false
+---
+
+**Summary: SIG-etcd has patched another potential issue blocking upgrades from v3.5 to v3.6.  If you are upgrading, make sure to update to [v3.5.26] or later first.**
+
+## Issue Summary
+
+Recently, the etcd community addressed an issue that may appear when users upgrade from v3.5 to v3.6.  This bug can cause the cluster to report "zombie members", which are etcd nodes that were removed from the database cluster some time ago, and are re-appearing and joining database consensus.  The etcd cluster is then inoperable until these zombie members are removed.
+
+In etcd v3.5 and earlier, the v2store was the source of truth for membership data, even though the v3store was also present. As a part of our [v2store deprecation plan], in v3.6 the v3store is the source of truth for cluster membership. Through a [bug report] we found out that, in some older clusters, v2store and v3store could become inconsistent.  This inconsistency manifests after upgrading as seeing old, removed "zombie" cluster members re-appearing in the cluster.
+
+## The Fix and Upgrade Path
+
+We’ve added a [mechanism in etcd v3.5.26] to automatically sync v3store from v2store, ensuring that affected clusters are repaired before upgrading to 3.6.x.
+
+With many users upgrading etcd as part of upgrading to Kubernetes v1.35 (releasing this week), we have provided the following safe upgrade path:
```

Comment:
I suggest that we don't mention Kubernetes at all to avoid any possible confusion. This issue has nothing to do with Kubernetes. Also note that we already bumped to etcd v3.6 in Kubernetes 1.34.